cdk/Pipfile

```diff
@@ -4,24 +4,27 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+"aws-cdk.core" = "~=1.105.0"
 "aws-cdk.aws-s3" = "~=1.105.0"
+"aws-cdk.aws-ec2" = "~=1.105.0"
+"aws-cdk.aws-ecr" = "~=1.105.0"
+"aws-cdk.aws-eks" = "~=1.105.0"
+"aws-cdk.aws-efs" = "~=1.105.0"
+"aws-cdk.aws-iam" = "~=1.105.0"
 "aws-cdk.aws-lambda" = "~=1.105.0"
-"aws-cdk.core" = "~=1.105.0"
 "aws-cdk.aws-kms" = "~=1.105.0"
-pyyaml = "*"
-"aws-cdk.aws-eks" = "~=1.105.0"
-requests = "*"
 "aws-cdk.aws-backup" = "~=1.105.0"
-"aws-cdk.aws-iam" = "~=1.105.0"
-"aws-cdk.aws-ecr" = "~=1.105.0"
+requests = "~=2.25.1"
```

Comment:
Not sure why some packages are in "" and some are not. 

This list better be sorted.

Reply:
I think it does it for packages that have the "." in them.

Reply:
No. "" is not required for the package names. I have tried and all works without "". Please make consistent.

Reply:
Per slack conversation: just delete Pipfile and Pipfile.lock since we have all in setup.py and it is called anyway when doing bla bla requirements.txt

Reply:
Please sort name list in setup.py

Reply:
Works for me.